cmd/init.go

```diff
@@ -48,18 +48,27 @@ func newInitCmd(rootConfig *RootCommandConfig) *cobra.Command {
 	// initCmd represents the init command
 	var initCmd = &cobra.Command{
 		Use:   "init [stack] or [repository]/[stack] [template]",
-		Short: "Initialize an Appsody project with a stack and template app",
-		Long: `This creates a new Appsody project in a local directory or sets up the local dev environment of an existing Appsody project.
-
-If the [repository] is not specified the default repository will be used. If no [template] is specified, the default template will be used.
-With the [stack], [repository]/[stack], [stack] [template] or [repository]/[stack] [template] arguments, this command will setup a new Appsody project. It will create an Appsody stack config file, unzip a template app, and run the stack init script to setup the local dev environment. It is typically run on an empty directory and may fail
-if files already exist. See the --overwrite and --no-template options for more details.
-Use 'appsody list' to see the available stack options.
-
-If keyword "none" is specified instead of a [template], the project will be initialized to use Appsody, and no template will be provided.
-
-Without the [stack] argument, this command must be run on an existing Appsody project and will only run the stack init script to
-setup the local dev environment.`,
+		Short: "Initialize an Appsody project.",
+		Long: `Set up the local Appsody development environment. You can do this for an existing project or use the template application provided by the stack. 
+
+By default, the command creates an Appsody stack configuration file and provides a simple default application. You can also choose to initialize a project with a different template application, or no template at all. 
+
+If you run this command in a directory that is not empty, and you want to initialise a project with a template application, the command fails unless you specify the "overwrite" option [--overwrite].
```

Comment:
initialize (US spelling)

Reply:
This sentence is back to front, we need something like...
"To initialize a project with a template application, in a directory that is not empty, you need to specify the overwrite option."